src/PrestaShopBundle/Form/Validator/Constraints/MultipleEmailsWithSeparatorValidator.php

```diff
@@ -89,4 +83,21 @@ public function validate($value, Constraint $constraint)
             ;
         }
     }
+
+    protected function getInvalidMailsMessage(array $invalidEmails): string
+    {
+        if (count($invalidEmails) > 1) {
+            return $this->translator->trans(
+                'Invalid emails : %invalid_emails%.',
```

Comment:
```suggestion
                'Invalid emails: %invalid_emails%.',
```
One last thing because there's no space before colons in English. Thank you :-)